src/ems_client.js

```diff
@@ -111,6 +110,10 @@ export class EMSClient {
       console.warn('The "kbnVersion" parameter for ems-client is deprecated. Please use "appVersion" instead.');
       appVersion = appVersion || kbnVersion;
     }
+    
+    if (!fetchFunction) {
+      console.warn('`fetchFunction` not provided. Defaulting to standard Javascript fetch');
```

Comment:
this will break server-side, since `fetch` doesn't exist (See @nickpeihl ). instead, throw.

Reply:
Ok, crossed paths in our comments. We're on the same page. I'll update it